No PMIx server was reachable, but a PMI1/2 was detected.
If srun is being used to launch application,  4 singletons will be started.
Traceback (most recent call last):
  File "/scratch/pzz1/3d-1d/sweeper_parallel.py", line 26, in <module>
    import fem
  File "/scratch/pzz1/3d-1d/./src/fem/__init__.py", line 1, in <module>
    from .sink import Sink
  File "/scratch/pzz1/3d-1d/./src/fem/sink.py", line 2, in <module>
    from tissue import AveragingRadius
  File "/scratch/pzz1/3d-1d/./src/tissue/__init__.py", line 3, in <module>
    from .domain_build import DomainBuild
  File "/scratch/pzz1/3d-1d/./src/tissue/domain_build.py", line 2, in <module>
    from dolfin import SubDomain, MeshFunction, Measure, UnitCubeMesh, facets, near, UserExpression
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dolfin/__init__.py", line 34, in <module>
    from .cpp import __version__
ImportError: /opt/apps/xalt/default/lib64/libcrypto.so: version `OPENSSL_3.3.0' not found (required by /home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dolfin/../../.././././libssl.so.3)
srun: error: bb3u23c1: task 0: Exited with exit code 1
/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/block/__init__.py:15: UserWarning: The cbc.block repository has moved to https://github.com/blocknics/cbc.block
  warnings.warn('The cbc.block repository has moved to https://github.com/blocknics/cbc.block', UserWarning)
/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/block/__init__.py:15: UserWarning: The cbc.block repository has moved to https://github.com/blocknics/cbc.block
  warnings.warn('The cbc.block repository has moved to https://github.com/blocknics/cbc.block', UserWarning)
/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/block/__init__.py:15: UserWarning: The cbc.block repository has moved to https://github.com/blocknics/cbc.block
  warnings.warn('The cbc.block repository has moved to https://github.com/blocknics/cbc.block', UserWarning)
Averaging over 224 cells:   0%|          | 0/224 [00:00<?, ?it/s]Averaging over 224 cells: 100%|██████████| 224/224 [00:00<00:00, 4306.48it/s]
Averaging over 224 cells:   0%|          | 0/224 [00:00<?, ?it/s]Averaging over 224 cells: 100%|██████████| 224/224 [00:00<00:00, 4298.54it/s]
Averaging over 224 cells:   0%|          | 0/224 [00:00<?, ?it/s]Averaging over 224 cells: 100%|██████████| 224/224 [00:00<00:00, 4250.59it/s]
Traceback (most recent call last):
  File "/scratch/pzz1/3d-1d/sweeper_parallel.py", line 197, in <module>
    worker_init(TEST_DOMAIN, LOWER_BOUNDS, UPPER_BOUNDS)
  File "/scratch/pzz1/3d-1d/sweeper_parallel.py", line 79, in worker_init
    SOLVER.solve(
  File "/scratch/pzz1/3d-1d/./src/fem/sub_cubes.py", line 40, in solve
    super().solve(gamma, gamma_a, gamma_R, mu, k_t, P_in, P_cvp)
  File "/scratch/pzz1/3d-1d/./src/fem/velo.py", line 14, in solve
    super().solve(gamma, gamma_a, gamma_R, mu, k_t, P_in, P_cvp)
  File "/scratch/pzz1/3d-1d/./src/fem/sink.py", line 62, in solve
    A, b = map(ii_assemble, (a_forms, L_forms))
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/xii/assembler/xii_assembly.py", line 62, in assemble
    blocks = reshape_list(list(map(assemble, flatten_list(form))), shape)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/xii/assembler/xii_assembly.py", line 49, in assemble
    tensor = module.assemble_form(form, arity)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/xii/assembler/average_assembly.py", line 38, in assemble_form
    return assembler.assemble(form, arity)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/xii/assembler/reduced_assembler.py", line 61, in assemble
    components.append(xii.assembler.xii_assembly.assemble(Form([integral])))
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/xii/assembler/xii_assembly.py", line 55, in assemble
    return df.assemble(form)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dolfin/fem/assembling.py", line 198, in assemble
    dolfin_form = _create_dolfin_form(form, form_compiler_parameters)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dolfin/fem/assembling.py", line 56, in _create_dolfin_form
    return Form(form,
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dolfin/fem/form.py", line 43, in __init__
    ufc_form = ffc_jit(form, form_compiler_parameters=form_compiler_parameters,
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dolfin/jit/jit.py", line 47, in mpi_jit
    return local_jit(*args, **kwargs)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dolfin/jit/jit.py", line 97, in ffc_jit
    return ffc.jit(ufl_form, parameters=p)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/ffc/jitcompiler.py", line 217, in jit
    module = jit_build(ufl_object, module_name, parameters)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/ffc/jitcompiler.py", line 130, in jit_build
    module, signature = dijitso.jit(jitable=ufl_object,
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dijitso/jit.py", line 165, in jit
    header, source, dependencies = generate(jitable, name, signature, params["generator"])
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/ffc/jitcompiler.py", line 76, in jit_generate
    dep_module_name = jit(dep, parameters, indirect=True)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/ffc/jitcompiler.py", line 217, in jit
    module = jit_build(ufl_object, module_name, parameters)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/ffc/jitcompiler.py", line 130, in jit_build
    module, signature = dijitso.jit(jitable=ufl_object,
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dijitso/jit.py", line 177, in jit
    build_shared_library(signature, header, source, dependencies,
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dijitso/build.py", line 181, in build_shared_library
    lockfree_move_file(temp_src_filename, src_filename)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dijitso/system.py", line 248, in lockfree_move_file
    return _lockfree_move_file(src, dst, False)
  File "/home/pzz1/.conda/envs/cmor_mdanderson/lib/python3.10/site-packages/dijitso/system.py", line 327, in _lockfree_move_file
    raise RuntimeError("Destination file should exist at this point!")
RuntimeError: Destination file should exist at this point!
srun: error: bb3u23c1: task 2: Exited with exit code 1
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** STEP 1074105.0 ON bb3u23c1 CANCELLED AT 2025-04-22T23:52:44 ***
slurmstepd: error: *** JOB 1074105 ON bb3u23c1 CANCELLED AT 2025-04-22T23:52:44 ***
